pandas/tests/groupby/test_api.py

```diff
@@ -119,11 +119,11 @@ def test_all_methods_categorized(mframe):
 Was a new method recently added?
 
 Every public method On Grouper must appear in exactly one the
-following three lists defined in pandas.core.groupby.base:
+following three lists defined in pandas._core.groupby.base:
 - `reduction_kernels`
 - `transformation_kernels`
 - `groupby_other_methods`
-see the comments in pandas/core/groupby/base.py for guidance on
+see the comments in pandas/_core.groupby/base.py for guidance on
```

Comment:
```suggestion
see the comments in pandas/_core/groupby/base.py for guidance on
```

Looks like there's one below (and possibly elsewhere)?

Reply:
Thanks - my script for automated changes was doing 

```
old = f'core.{submodule}'
new = f'_core.{submodule}'
```

whereas it should have been `\.` in the regex. Will fix.

Reply:
This is now fixed.